x-pack/test/api_integration/apis/infra/log_entry_highlights.ts

```diff
@@ -121,9 +121,7 @@ export default function({ getService }: FtrProviderContext) {
             expect(logEntryHighlight.columns).to.have.length(3);
             expect(logEntryHighlight.columns[1]).to.have.property('field');
             expect(logEntryHighlight.columns[1]).to.have.property('highlights');
-            expect(logEntryHighlight.columns[1].highlights).to.eql([
-              'generate_test_data/simple_logs',
-            ]);
+            expect(logEntryHighlight.columns[1].highlights).to.eql(['generate_test_data']);
```

Comment:
i'm not sure what's happening here, @Kerry350 can you point me where i would need to look for these changes?

Reply:
cc @elastic/logs-metrics-ui

Reply:
The expectation is that this echoes the query from line 104, so it shouldn't change. :thinking: I don't see a connection to doc types.

Reply:
Maybe https://github.com/elastic/elasticsearch/issues/29561?  I dug into this further, it's somewhere at the query level.  UI is highlighting as expected.  Mappings LGTM with event.dataset as keyword.

Reply:
@elastic/logs-metrics-ui, any assistance we can get here would be great. There is currently quite a bit blocked by this PR. Thanks!

Reply:
This returns as expected:
```
GET filebeat-simple-logs/_search
{
  "query": {
    "multi_match": {
      "query": "generate_test_data/simple_logs",
      "type": "phrase"
    }
  },
  "highlight": {
    "fields": {
      "event.dataset": {}
    }
  }
}
```

Reply:
Narrowed it down to `fragment_size: 1`

Reply:
Instead of merging this change with unknown consequences, let's revert it and skip the test, creating a blocker issue for 8.0.

Reply:
Sorry for the delay. Agreed - we'll look at the highlighting change separately.